package.json

```diff
@@ -279,7 +286,9 @@
                             "host": "${promptForHost}",
                             "password": "${promptForPassword}",
                             "rootDir": "^\"\\${workspaceFolder}\"",
-                            "enableDebuggerAutoRecovery": false
+                            "enableDebuggerAutoRecovery": false,
+                            "enableAutoTerminate": false,
```

Comment:
Change this to 'stopDebuggerOnAppExit' like the comment above?

Reply:
done